### ISSUE TITLE
[FTX] Can't reconnect on fills (authenticated) / (connect OK, and even Ping pong coded)

### ISSUE THREAD

**Issue by dgosset** [knowm/XChange#4116]
Hi everyone,

My issue is not really an issue because i don't push my version with the authenticated fills by weboscket for FTX.

But, is there some peoples on fills websocket with FTX? I run it with succes, but some times i'm disconnect and resubscribe can't reconnect. It tried, but FTX reply with a 400 "Not logged in". 

I'm on GCP, Appengine, 1 instance. So it don't think it is a mutliple instance issue on websocket (was my firt idea)
DOc FTX say "One websocket connection may be logged in to at most one user. If the connection is already authenticated, further attempts to log in will result in 400s."

I'm coded pinp/pong like it is in "okcoin module xchange". It works. But don't fix my issue :(

Some time, FTX disconnect me (maybe because i don't have any orders on the marketplace? But i don't see any rules in doc)

cc @makarid : you are on FTX, right? what do you think about it?

Regards.

**Comment by makarid** [comment]
Yes, my implementation does the following, i instantiate the websocket and calling the "fills" on the FtxStreamingTradeService constructor and not on the function getUserTrades.

**Comment by dgosset** [comment]
@makarid Can you share me you FtxStreamingTradeService  with https://gist.github.com ?

**Comment by makarid** [comment]
Copy paste it from the pull request

Στις Τετ, 9 Ιουν 2021 στις 6:24 μ.μ., ο/η Denis ***@***.***>
έγραψε:

**Comment by elkceo** [comment]
@dgosset Can you share with your code how you getting orderChanges?
